### DISCUSSION TITLE
Needs

### DISCUSSION THREAD

**Discussion by woolcake** [eneshecan/whatsapp-for-linux#274] [General]
Cambiar el tipo y tamaño de la fuente, responsive (no se si lo sea pero no se adapta bien a mi pantalla, parace que tuviera márgenes que es causado por el zoom, el zoom no es responsive), además tiene un bug donde las palabras se mueven de arriba a bajo como si estuviera vibrando.

**Comment by xeco23** [comment]
If I understand you correctly, we can introduce this option in the Preferences window to disable zooming all the elements except the text: [zoom-text-only](https://webkitgtk.org/reference/webkit2gtk/stable/property.Settings.zoom-text-only.html) to resolve this issue. Please create an issue for this.
